docs/UserGuide.md

```diff
@@ -223,6 +223,7 @@ Format: `delete INDEX`
 * Deletes the patient at the specified `INDEX`.
 * The index refers to the index number shown in the displayed patient records list.
 * The index must be a positive integer 1, 2, 3, …​
+* The index must be within the [system limit](#faq).
```

Comment:
Does the app still have an inconsistent error message above the limit? Since we're trying to address issue #142.

Reply:
Yes it does, but if the index provided exceeds the system limit, in my opinion, it might be okay to use the `Invalid Command Format` error instead of the `Invalid Index` error.